src/Main.elm

```diff
@@ -73,6 +84,8 @@ type Msg
     | ChangedUrl Url.Url
     | ClickedLink Browser.UrlRequest
     | Now Posix
+    | CheckViewport Posix
```

Comment:
Why does this need a Posix?
```suggestion
    | CheckViewport
```

Reply:
Yeah this is a weird one. The main `Cmd.batch` that gets the viewport and current time needs both items in the list you give it to be the same type. So this was a quick and dirty way to get it working. I figured you would know of a better way to handle it.